From 3d357ab5dee6f08b11d8afa01d95248040409eec Mon Sep 17 00:00:00 2001
From: Sebastien Boeuf <sebastien.boeuf@intel.com>
Date: Fri, 27 Mar 2015 15:44:22 +0100
Subject: [PATCH 2/3] Change VCM driver absolute default position

This patch modifies default value in dw9714 driver because this is the rest
value for the module. However, it would be better to read this value from
BIOS instead of harcoding it, because if we have different camera modules
with different init values for VCMs, we don't want this value to be hardcoded
in the driver. We take this for now but it should be updated when ACPI 5.1
capabilities will be integrated in our kernel.

Issue: GMINL-6562
Change-Id: Id89b3743eae3cb622b55f3d5db8eda31c158b0bf
Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>
---
 drivers/media/i2c/atomisp/dw9714.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/atomisp/dw9714.c b/drivers/media/i2c/atomisp/dw9714.c
index 9698967..94d6471 100644
--- a/drivers/media/i2c/atomisp/dw9714.c
+++ b/drivers/media/i2c/atomisp/dw9714.c
@@ -254,7 +254,7 @@ struct dw9714_control dw9714_controls[] = {
 			.minimum = 0,
 			.maximum = DW9714_MAX_FOCUS_POS,
 			.step = 1,
-			.default_value = 0,
+			.default_value = DW9714_MAX_FOCUS_POS,
 			.flags = 0,
 		},
 		.tweak = dw9714_t_focus_abs,
-- 
1.9.1

